.changeset/angry-timers-juggle.md

```diff
@@ -0,0 +1,7 @@
+---
+"@sei-js/cosmjs": patch
+"@sei-js/proto": patch
```

Comment:
think this might need to be minor/major for proto - unless there's no breaking changes?